src/worker/token.worker.ts

```diff
@@ -67,7 +71,12 @@ const messageHandler = async ({
 
     if (!response) {
       // If the request times out, abort it and let `fetchWithTimeout` raise the error.
-      abortController.abort();
+      if (abortController) abortController.abort();
+
+      port.postMessage({
+        error: "Timeout when executing 'fetch'"
+      });
```

Comment:
Why are we adding this?

Reply:
This was mostly for the benefit of the test, to be able to determine that the failure was because of a timeout, but could also be a useful error message on the client regardless. What do you think?

Reply:
I think this is fine. It rejects the promise using the same error we already have incase it timesout based on the setTimeout.

Reply:
It shouldn't do.. if the timeout is hit, no response will be returned from `Promise.race` and falls into the next block that checks for an empty response, rather than falling into the catch. I could be mistaken; I tried to test for that here: https://github.com/auth0/auth0-spa-js/blob/fix/abort-controller/__tests__/token.worker.test.ts#L146